frontend/src/metabase/ui/theme.ts

```diff
@@ -38,7 +38,7 @@ export const getThemeOverrides = (): MantineThemeOverride => ({
   primaryColor: "brand",
   primaryShade: 0,
   shadows: {
-    sm: "0px 4px 4px 0px rgba(0, 0, 0, 0.08)",
+    sm: "0px 1px 4px 2px rgba(0, 0, 0, 0.08)",
```

Comment:
Adjust shadow styling to match the designs styling. The other shadow styling did not create a shadow on the top of the popover making it blend in with the dashboard header